docs/src/components/home/sections/AuthenticationSection.tsx

```diff
@@ -0,0 +1,195 @@
+  react: `import { Amplify } from 'aws-amplify';
+
+import { Authenticator } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+import awsExports from './aws-exports';
+Amplify.configure(awsExports);
+
+export default function App() {
+  return (
+    <Authenticator>
+      {({ signOut, user }) => (
+        <main>
+          <h1>Hello {user.username}</h1>
+          <button onClick={signOut}>Sign out</button>
+        </main>
+      )}
+    </Authenticator>
+  );
+}`,
+  vue: `<script setup>
+  import { Authenticator } from "@aws-amplify/ui-vue";
+  import "@aws-amplify/ui-vue/styles.css";
+
+  import Amplify from 'aws-amplify';
+  import awsconfig from './aws-exports';
+
+  Amplify.configure(awsconfig);
+</script>
+
+<template>
+  <authenticator>
+    <template v-slot="{ user, signOut }">
+      <h1>Hello {{ user.username }}!</h1>
+      <button @click="signOut">Sign Out</button>
+    </template>
+  </authenticator>
+</template>`,
+  flutter: `
+//...
+class _MyAppState extends State<MyApp> {
+  @override
+  void initState() {
+    super.initState();
+    _configureAmplify();
+  }
+
+  void _configureAmplify() async {
+    try {
+      await Amplify.addPlugin(AmplifyAuthCognito());
+      await Amplify.configure(amplifyconfig);
+      print('Successfully configured');
+    } on Exception catch (e) {
+      print('Error configuring Amplify: $e');
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Authenticator(
+      child: MaterialApp(
+        builder: Authenticator.builder(),
+        home: const Scaffold(
+          body: Center(
+            child: Text('You are logged in!'),
+          ),
+        ),
+      ),
+    );
+  }
+}`,
+};
+
+const languages = {
+  react: 'jsx',
+  angular: 'javascript', // is this the best primsa language?
+  vue: 'javascript',
+  flutter: 'dart',
+};
+
+const fileName = {
+  react: 'index.tsx',
+  angular: 'index.html',
+  vue: 'index.vue',
+  flutter: 'main.dart',
+};
+
+export const AuthenticationSection = ({ platform }) => {
+  const ref = useRef<HTMLDivElement | null>(null);
+  const entry = useIntersectionObserver(ref, {
+    threshold: 0.125,
+    freezeOnceVisible: true,
+  });
+  const isVisible = !!entry?.isIntersecting;
+  const hiddenOnMobile = useBreakpointValue({
+    base: false,
+    medium: true,
+  });
+
+  return (
+    <View
+      ref={ref}
+      as="section"
+      id="authentication"
+      className={classNames(
+        'docs-home-section',
+        'fade-in',
+        isVisible && 'shown'
+      )}
+    >
+      <Flex direction="column">
+        <Flex direction="column" className="docs-home-subsection--thin">
+          <Heading level={2}>
+            <strong>Authentication</strong> made easy
+          </Heading>
+          <Text className="docs-home-text">
+            Add authentication to your app in under 10 lines of code using the
+            Authenticator component. The Authenticator works seamlessly with the{' '}
```

Comment:
Can we add a link to the Authenticator page here? "Add authentication to your app in under 10 lines of code using the [Authenticator component](https://ui.docs.amplify.aws/react/components/authenticator)."